pylint/checkers/similar.py

```diff
@@ -427,7 +427,10 @@ def reduce_map_data(cls, linter, data):
         """Reduces and recombines data into a format that we can report on
 
         The partner function of get_map_data()"""
-        recombined = SimilarChecker(linter)
+        try:
+            recombined = linter._checkers["similarities"][0]
+        except:
```

Comment:
Bare except.

Try running the following inside your checkout.
`tox -e formatting`